pkg/controller/route/resources/virtual_service.go

```diff
@@ -142,6 +144,10 @@ func makeVirtualServiceRoute(domains []string, ns string, targets []traffic.Revi
 	route := v1alpha3.HTTPRoute{
 		Match: matches,
 		Route: weights,
+		Timeout: DefaultRouteTimeout,
+		AppendHeaders: map[string]string{
+			IstioTimeoutHackHeaderKey:   IstioTimeoutHackHeaderValue,
+		},
 	}
 	// Add traffic rules for activator.
 	return addActivatorRoutes(&route, ns, inactive)
```

Comment:
Instead of having two timeout constants of 60s, and having one overwrite another, can we switch to always using the 60 second timeout?

Reply:
Done